test/decorators/project_decorator_test.rb

```diff
@@ -30,16 +38,18 @@ class ProjectDecoratorTest < ActiveSupport::TestCase
         [:reviews, 'Ratings & Reviews', "/p/#{linux.vanity_url}/reviews/summary"],
         [:map, 'User & Contributor Locations', "/p/#{linux.vanity_url}/map"]
       ]
-    ]
+    ].tap do |menus|
+      linux.decorate.append_sbom_menu(menus, current_user)
+    end
   end
 
   describe 'sidebar' do
     it 'should contain 4 sections' do
-      _(linux.decorate.sidebar.length).must_equal 4
+      _(linux.decorate.sidebar(current_user).length).must_equal 4
     end
 
     it 'should return projects menu list' do
-      _(linux.decorate.sidebar).must_equal sidebar
+      _(linux.decorate.sidebar(current_user)).must_equal sidebar
```

Comment:
This test is incorrect. Think about what we are doing here.
On the left hand side, we are creating a sidebar object with the same code that we see in the ProjectDecorator. On the right hand side, we call similar code in ProjectDecorator.
The original test was also incorrect, so this is not something you need to deal with in this PR. This is just to be aware of bad patterns in tests.